### ISSUE TITLE
update Explore view tags

### ISSUE THREAD

**Issue by SimonLab** [emfoundation/ce100-app#1090]
- [ ] Display tags from the API
- [ ] Only dispaly tags that are not deprecated

@asimonw I'm not sure yet if the deprecated property for the tags and countries is implemented yet?

![image](https://user-images.githubusercontent.com/6057298/32941718-9a3deb18-cb7e-11e7-89ce-aabb614afbfe.png)

**Comment by asimonw** [comment]
@SimonLab Good point, that's not yet in place. Would it be ok to implement it without this feature for now? Or would it require rework to add it at a later date?

@georgemillard Another field to be added to our `Tag` model ;)

**Comment by SimonLab** [comment]
it's not a blocker at the moment and I don't think it will be too complicated to later on filter the deprecated tag. I juts wanted to be sure that I didn't miss this. I'll keep this issue open as a reminder.

**Comment by asimonw** [comment]
@SimonLab Thanks. I put it on the radar on our side, shouldn't take too long to add.

A tiny question: can the `HEADQUARTERS/LOCATION ...` parent tag just be `LOCATION` like in production? (I appreciate that this is a work in progress screen shot, but just be be sure.)
